cmd/thanos/receive.go

```diff
@@ -511,8 +509,10 @@ func setupHashring(g *run.Group,
 				if c == nil {
 					webHandler.Hashring(receive.SingleNodeHashring(conf.endpoint))
 				}
+				level.Info(logger).Log("msg", "Empty hashring config. Set up single node hashring.")
 
 				webHandler.Hashring(receive.NewMultiHashring(algorithm, conf.replicationFactor, c))
+				level.Info(logger).Log("msg", "Set up hashring for the given hashring config.")
```

Comment:
Do we need the logs in L512 and L515? I feel having them are too verbose. Maybe keep only 1 or make them debug level

Reply:
Yeah I kind of copied them from [here](https://github.com/thanos-io/thanos/pull/5777/files#diff-3d45f9beb5b670e94fe2715e21477a3fb8f5db7bf97f01cddf6e108bb98761a6L463) and [here](https://github.com/thanos-io/thanos/pull/5777/files#diff-3d45f9beb5b670e94fe2715e21477a3fb8f5db7bf97f01cddf6e108bb98761a6L482). If you think users don't need both, I'll remove 1.

Reply:
Ah, maybe now I fully understand what you meant. Back then I forgot the `else` statement, so both of these logs would show up^^

Now I added the `else` statement, only 1 of them will show up. Does sth like this look ok for you?
				if c == nil {
					webHandler.Hashring(receive.SingleNodeHashring(conf.endpoint))
level.Info(logger).Log("msg", "Empty hashring config. Set up single node hashring.")
				} else {
					webHandler.Hashring(receive.NewMultiHashring(algorithm, conf.replicationFactor, c))
                                        level.Info(logger).Log("msg", "Set up hashring for the given hashring config.")
				}

Reply:
> Ah, maybe now I fully understand what you meant. Back then I forgot the `else` statement, so both of these logs would show up^^
> 
> Now I added the `else` statement, only 1 of them will show up. Does sth like this look ok for you?
> 
> ```
> if c == nil {
>     webHandler.Hashring(receive.SingleNodeHashring(conf.endpoint))
>     level.Info(logger).Log("msg", "Empty hashring config. Set up single node hashring.")
> } else {
>     webHandler.Hashring(receive.NewMultiHashring(algorithm, conf.replicationFactor, c))
>     level.Info(logger).Log("msg", "Set up hashring for the given hashring config.")
> }
> ```

@yeya24 I did this in my latest commit. Let me know if you think this works.